server/update-user.js

```diff
@@ -66,7 +66,7 @@ export default function updateUserFactory(analyticsClient) {
       synchronized_segments.length > 0 && // Should we move to "Send no one by default ?"
       !_.intersection(segment_ids, synchronized_segments).length
       ) {
-      hull.logger.info("outgoing.user.skip", { ...loggingProperties, reason: "not matching any segment", segment_ids, traits });
+      hull.asUser(loggingProperties).logger.info("outgoing.user.skip", { reason: "not matching any segment", segment_ids, traits });
```

Comment:
@mkawalek btw. since we are on the same scope level with couple for `hull.asUser` calls maybe let's init it once as a const in place of `loggingProperties`?